docker-compose.yml

```diff
@@ -1,24 +1,46 @@
 services:
   database:
-      image: postgres
-      restart: always
-      networks:
-        metaploy-private-network:
-          aliases:
-            - postgres-of-babel
-      environment:
-        - POSTGRES_USER=${DATABASE_USERNAME}
-        - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
-        - POSTGRES_DB=${DATABASE_NAME}
-        - PGPORT=${DATABASE_PORT}
-        - PGHOST=${DATABASE_HOST}
-      volumes:
-        - db-volume:/var/lib/postgresql/data
+    image: postgres
```

Comment:
tag version of postgres being used.

Reply:
we are using the latest version. shouldnt be a problem in the future i think.